.vib/influxdb/goss/vars.yaml

```diff
@@ -0,0 +1,14 @@
+auth:
+  admin:
+    token: "4dm1nT0k3n"
+  user:
+    username: "testUser"
```

Comment:
I have not used `username: user` as using a non-generic value simplifies the process of obtaining the token in the tests.